omod/src/main/webapp/portlets/flagPatientOverview.jsp

```diff
@@ -1,8 +1,22 @@
+<%@page import="org.openmrs.module.patientflags.Flag"%>
 <%@ include file="/WEB-INF/template/include.jsp"%>
 
 <openmrs:require privilege="View Patients" otherwise="/login.htm"
 	redirect="/index.htm" />
 
 <c:forEach items="${model.flags}" var="flag">
-	<span ${flag.priority != null ? flag.priority.style : ''}>${flag.localizedMessage}</span><br />
+	<span ${flag.priority.style}> 

```

Comment:
Is JSP null safe?  Priority is not a required property of a flag, so won't this through a NPE if flag.priority = null?   Maybe I'm wrong, I haven't been working with JSP in a bit.